src/middlewares/validators/claim.validator.ts

```diff
@@ -77,6 +77,56 @@ export const claimPostSchema = Joi.object({
   }),
 });
 
+export const credentialPostSchema = Joi.object({
+  context: Joi.array().required(),
```

Comment:
hey remove required from ALL of these please except the id

Reply:
Note i think we might want to use the credential endpoint for OTHER types of credentials

Are we linking somewhere to the schema?  its in the credential

    "https://www.w3.org/2018/credentials/v1",
    "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.3.json",
    
    can we please REMOVE the Joi validation
    
    and later we can add credential validation using the credential schema, which is the correct way to do it.

Reply:
we do NOT want to define the schema here in hardcode AND in the real schema itself

so lets remove here